system/unique_board_id/unique_board_id.c

```diff
@@ -24,9 +24,13 @@ int main() {
     pico_unique_board_id_t board_id;
     pico_get_unique_board_id(&board_id);
 
-    printf("Unique identifier:");
-    for (int i = 0; i < PICO_UNIQUE_BOARD_ID_SIZE_BYTES; ++i) {
-        printf(" %02x", board_id.id[i]);
-    }
-    printf("\n");
+    while(1) {
+   		printf("Unique identifier:");
+    	for (int i = 0; i < PICO_UNIQUE_BOARD_ID_SIZE_BYTES; ++i) {
```

Comment:
Can you sort out the whitespace too please? (use 4-spaces-per-indent to match existing code, don't use tabs)